### PR TITLE
Show durations as minutes in trends and history graphs

### LINKED ISSUE #2184: Histograms/Trends: Duration display as seconds instead of minutes
**Current Behavior**
Hello, when you look at your Duration in **Histograms** or **Trends** the labels, median and tooltips are only display as seconds making it difficult to quickly read and understand the values. 

**Expected behavior/code**
A second line in the label, or simply replaced with the converted values to minutes would be welcome.
(38:53 instead of 2312)

**Environment**
- Browser Version: Brave (0.67.125) based on Chromium: 76.0.3809.100
- OS: Windows 10

Regards,

### PR DESCRIPTION
**Pull request by bdbch**
This should fix issue #2184 while keeping the formatting in the render function of the specific tooltip / graph.

fixes #2184

### CODE REVIEW

File: src/components/Player/Pages/Histograms/Histograms.jsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React from 'react';
```
Comment (bdbch) [inline src/components/Player/Pages/Histograms/Histograms.jsx]: FYI: Just my formatting addon sorting imports. if I should change it back let me know.

**Comment by howardchung** [comment]
LGTM, but some minor lint issues to clean up?

**Comment by bdbch** [comment]
Will fix linting issues

**Comment by bdbch** [comment]
Done, could be merged if okay for you. @howardchung

**Pull request merged**